Sources/WebDriver/Requests.swift

```diff
@@ -524,6 +524,41 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidlocation
+    public enum SessionGeoLocation {
```

Comment:
Let's stick to the name from the spec, even if GeoLocation is more descriptive.
```suggestion
    public enum SessionLocation {
```